File: semester I/Praktikum Pemrograman I/Modul5_JovanGilbertNatamasindah_2310817310002/Python/PRAK501-2310817310002-JovanGilbertNatamasindah.py
def MaxBilangan(x,y,i,j) :

    if (x >= y and x >= i and x >= j) :
        return x
    
    elif (y >= x and y >= i and y >= j) :
        return y

    elif (i >= x and i >= y and i >= j) :
        return i

    elif (j >= x and j >= y and j >= i) :
        return j


a, b, c, d = map(int,input("Masukkan bilangan - bilangan : ").split(" "))
Bilangan_max = MaxBilangan(a,b,c,d)
print("Bilangan Maksimal : ",Bilangan_max)

File: semester I/Praktikum Pemrograman I/Modul5_JovanGilbertNatamasindah_2310817310002/C/PRAK501-2310817310002-JovanGilbertNatamasindah.c
#include <stdio.h>

int MaxBilangan(int x, int y, int i, int j) {

    if (x >= y && x >= i && x >= j) {
        return x;
    }

    else if ( y >= x && y >= i && y >= j) {
        return y;
    }

    else if ( i >= x && i >= y && i >= j ) {
        return i;
    }

    else if (j >= x && j >= y && j >= i) {
        return j;
    }
}

int main() {

    int a, b, c, d; 
    printf("Masukkan bilangan - bilangan : ");
    scanf("%d %d %d %d", &a, &b, &c, &d);

    int hasil = MaxBilangan(a, b, c, d);
    printf("Bilangan Maksimal : %d", hasil); 

return 0;
}









